src/TodoListItem.js

```diff
@@ -1,6 +1,5 @@
 import React from "react";
-import { useState } from "react";
 
-export default function TodoListItem(props) {
-  return <li>{props.todo}</li>;
+export default function TodoListItem({ key, todo }) {
```

Comment:
this is a little weird, but when we pass the `key` prop to the items which are `map`ed and rendered, react applies the `key` behind the scenes, but then sortof "swallows" it, so it never actually is passed to the component. 

so in this case and any other time you pass `key` to a component, you can remove it from the props destructuring, because it won't actually be defined

Reply:
<img width="960" alt="Screen Shot 2023-07-07 at 11 35 58 AM" src="https://github.com/evaw277/react-todo/assets/12771651/3fc0c68f-c38f-4c0a-b602-6685db2465f4">

the browser console will also warn us if we try to access the `key` prop

Reply:
if we need to have access to that `id` later on the todo item, we can pass it as a duplicate prop like `id`